roles/satosa/files/filebeat.yml

```diff
@@ -0,0 +1,6 @@
+---
+- paths:
+    - /opt/satosa/satosa-env/var/custom.log
```

Comment:
Where is the journald output from satosa (ie, the output from `journalctl -u satosa`) logged?  Or deos that also end up in syslog, and is it picked up from there?

Reply:
Satosa generic loggig will eventually lift on generic syslog logging, which is not enabled atm.